pkg/enum/docker.go

```
@@ -0,0 +1,320 @@
+	content, err := io.ReadAll(r)
+	if err != nil {
+		return nil, false, err
+	}
+	if maxSize > 0 && int64(len(content)) > maxSize {
+		return nil, false, nil
+	}
+	return content, true, nil
+}
+
+func isRegularTarFile(hdr *tar.Header) bool {
+	return hdr.Typeflag == tar.TypeReg || hdr.Typeflag == tar.TypeRegA
```

golangci-lint (2.12.2)

[error] SA1019: tar.TypeRegA has been deprecated since Go 1.11 and an alternative has been available since Go 1.1: Use TypeReg instead. 

(staticcheck)